knative-operator/deploy/resources/quickstart/serverless-application-quickstart.yaml

```diff
@@ -0,0 +1,215 @@
+
+          From the **Topology** view, look for your new application. Wait for
+          the build to complete. It may take a few minutes.
+
+          After the build completes, a green checkmark appears in the lower-left
+          corner of the service. Your application will say  “No Revisions” in
+          the center.
+
+          Do you see the completed application and build?
+      summary:
+        failed: Try the steps again.
+        success: You just created a Serverless app!
+      title: Creating a serverless application
+    - description: >
+        ### To see your application scale:
+
+        1. From the **Display Options** list at the top of the **Topology**
+        view, click **Pod Count**.
+
+        2. Wait for the Pod count to scale down to zero Pods. Scaling down may
+        take a few minutes.
+
+        3. Click the **Open URL** icon in the upper-right corner of the Knative
+        Service panel. The application opens in a new tab.
+
+        4. Close the new browser tab and return to the **Topology** view.
+
+        In the **Topology** view, you can see that your application scaled up to
+        one Pod to accommodate your request.  After a few minutes, your
+        application scales back down to zero Pods.
+      review:
+        failedTaskHelp: >-
+          This task isn’t verified yet. Try the task again, or [read
+          more](https://docs.openshift.com/container-platform/4.6/applications/application_life_cycle_management/odc-viewing-application-composition-using-topology-view.html#odc-scaling-application-pods-and-checking-builds-and-routes_viewing-application-composition-using-topology-view)
+          about this topic.
+        instructions: >-
+          #### To verify the application scaled down:
+
+          1. Click the revision inside your service. The badges under the Pod
+          ring and at the top of the side panel should be (REV).
+
+          2. Click the **Details** tab in the side panel.
+
+          Is the Pod ring autoscaled to zero?
+      summary:
+        failed: Try the steps again.
+        success: You just scaled up your application to accomodate a traffic request!
+      title: Demoing scalability
+    - description: >-
+        ### To connect an event source to your Knative Service:
+
+        1. On the **Topology** View, hover over your service with your cursor.
+        You should see a blue arrow.
+
+        2. Click and drag the blue arrow, then drop it anywhere outside the
+        service.
+
+        3. In the list that appears, click **Event Source**.
+
+        4. Under the **Type** field, click **PingSource**.
+
+        5. In the **Data** field, type `This message is from PingSource`. This
+        message is posted when the service is called.
+
+        6. In the **Schedule** field, type `* * * * *`.  This means that the
+        PingSource will make a call every minute.
+
+        7. In the **Application** field, select **Sample Serverless App**.
```

Comment:
s/`Sample Serverless App`/`django-ex-git-app` check my screenshot:
![image](https://user-images.githubusercontent.com/7945591/107236469-6aeeb200-6a2e-11eb-866c-53617f7669a0.png)

Reply:
Btw when I click: 
![image](https://user-images.githubusercontent.com/7945591/107237038-f9fbca00-6a2e-11eb-963f-e0b6f6b56134.png)
it leads me here where I create the source from scratch:
![image](https://user-images.githubusercontent.com/7945591/107237145-16980200-6a2f-11eb-960e-73860ed4eaeb.png)

Reply:
This is not exactly what is described on the panel on the right.